src/client/containers/NotificationBanner/index.tsx

```diff
@@ -0,0 +1,19 @@
+import { FC } from 'react';
+import { first } from 'lodash';
+
+import { useAppSelector, useAppDispatch } from '@hooks';
+import { NotificationsActions, NotificationsSelectors } from '@store';
+import { Banner } from '@components/common';
+
+export const NotificationBanner: FC = () => {
+  const latestMessage = first(useAppSelector(NotificationsSelectors.selectActiveMessages));
```

Comment:
is there a reason we need a container component instead of just having this all on the banner component?